### ISSUE TITLE
br/brotli encoding not supported on HtmlWeb

### ISSUE THREAD

**Issue by axtens** [zzzprojects/html-agility-pack#343]
### Description
Using HtmlWeb to LoadFromBrowser() from "https://aaronsremovals.com.au/contact-us/", I get a System.ArgumentException because the site is providing a header of "content-encoding: br" and HtmlWeb doesn't know what to do with it.

### Exception
```
System.ArgumentException
  HResult=0x80070057
  Message='br' is not a supported encoding name. For information on defining a custom encoding, see the documentation for the Encoding.RegisterProvider method.
Parameter name: name
  Source=mscorlib
  StackTrace:
   at System.Globalization.EncodingTable.internalGetCodePageFromName(String name)
   at System.Globalization.EncodingTable.GetCodePageFromName(String name)
   at System.Text.Encoding.GetEncoding(String name)
   at HtmlAgilityPack.HtmlWeb.Get(Uri uri, String method, String path, HtmlDocument doc, IWebProxy proxy, ICredentials creds)
   at HtmlAgilityPack.HtmlWeb.LoadUrl(Uri uri, String method, WebProxy proxy, NetworkCredential creds)
   at HtmlAgilityPack.HtmlWeb.Load(Uri uri, String method)
   at HtmlAgilityPack.HtmlWeb.Load(String url)
   at Sitelligence.Dossier.GetDocument() in C:\Users\bugma\Source\Repos\Sitelligence\Dossier.cs:line 444
   at Sitelligence.Dossier.Analyse(Boolean debug) in C:\Users\bugma\Source\Repos\Sitelligence\Dossier.cs:line 140
```

### Fiddle or Project
https://dotnetfiddle.net/25Vjsn but it won't run due to a security issue that I don't know how to address.

### Further technical details
- HAP version: 1.11.16
- .NET Framework 4.8
- [br encoding](https://en.wikipedia.org/wiki/Brotli)

**Comment by JonathanMagnan** [comment]
Hello @axtens ,

Thank you for reporting,

We will look at it.

Best Regards,

Jonathan

---

**Performance Libraries**
`context.BulkInsert(list, options => options.BatchSize = 1000);  `
[Entity Framework Extensions](http://entityframework-extensions.net/) • [Entity Framework Classic](http://entityframework-classic.net/) • [Bulk Operations](http://bulk-operations.net/) •  [Dapper Plus](http://dapper-plus.net/)

**Runtime Evaluation**
`Eval.Execute("x + y", new {x = 1, y = 2}); // return 3`
[C# Eval Function](http://eval-expression.net/) • [SQL Eval Function](http://eval-sql.net/)

**Comment by JonathanMagnan** [comment]
Hello @axtens 

My developer and me both tried the following code:

```csharp
var url = "https://aaronsremovals.com.au/contact-us/";
var web = new HtmlWeb();
var doc = web.LoadFromBrowser(url);
var text = doc.DocumentNode.InnerHtml;
```

but none of us got an error.

Do you know what we are missing here to reproduce the issue?

**Comment by JonathanMagnan** [comment]
If you get the error with our code, please save the source text into a file and provide us the file.

**Comment by JonathanMagnan** [comment]
Hello @axtens, 

Since our last conversation, we haven't heard from you. 

would it be possible to reproduce the issue ?

Best regards,

Jon

**Comment by axtens** [comment]
Did you get the email containing the paragraph: "I've made a project on
Github at <https://github.com/axtens/HAPBug>
<https://github.com/axtens/HAPBug> and have provided a test file called
"allstripped.com.au_urls.json". As said in the readme, this list of urls is
derived from their sitemap."?

Kind regards,
Bruce

**Comment by JonathanMagnan** [comment]
Hello @axtens ,

Today we released a new version v1.11.17 which improves how the encoding is solved (and it seems to be your issue)

Could you try it?

Otherwise, we will really need to have a working project with an `HTML` file or string. On our side, we tried on different computers but we get timeout issues with your projects even when waiting for one minute. And maybe something is returned differently in your country than ours.

**Comment by JonathanMagnan** [comment]
Hello @axtens,

Since we haven't heard back, we will close this issue.

Feel free to contact us once you can provide more information.
 
Best regards,

Jon
